src/docs/sdk/hub_and_scope_refactoring.mdx

```diff
@@ -164,6 +164,43 @@ This image illustrates the behavior of these new APIs and how scope data is appl
 
 For a zoomable version visit the <a href="https://miro.com/app/board/uXjVNtPiOfI=/?share_link_id=140058397661" target="_blank">Miro Board</a>
 
+Here's how the new APIs roughly map to the old-style APIs. In case of `configure_scope` and `push_scope` it's not clear-cut whether to use the current or the isolation scope. The choice essentially depends on how long/for what context the changed/forked scope should be active.
```

Comment:
```suggestion
Here's how the new APIs roughly map to the old-style APIs. In case of `configure_scope` and `push_scope` it's not clear-cut whether to use the current or the isolation scope. The choice essentially depends on how long/for which context the changed/forked scope should be active.
```